libs/store/src/lib/infrastructure/persistence/domain/state-handler.spec.ts

```diff
@@ -48,100 +47,137 @@ interface ReqDTO {
     uniqueName: IdentityKey;
     title: string;
     amount: number;
+    total: Money;
     lineItems: Array<LineItem>;
 }
 
 class Requisition extends BaseEntity<ReqDTO> {
     uniqueName: IdentityKey;
     title: string;
     amount: number;
+    total: Money;
```

Comment:
I am not sure how this this Requisition work and how we are using Parent BaseEntity

Reply:
I left the core things the same, so it uses the same approach as previously. 
A proxied entity relies on internal `composite.value` namespace.
`Proxy.get` trap tries to find the property in the `composite.value` namespace, if it’s not there then redirect to the default flow.
`Proxy.set` trap tries to set a new value in the `composite.value` namespace if possible, if not redirect to the default assignment process